portfolio/src/main/webapp/script.js

```diff
@@ -40,26 +37,48 @@ function addHobby() {
 function addProjectDescription(project) {
   
   const details = {
-    nlp: 'Building language models',
-    ta: 'Held weekly office hours and review sessions for introductory Data Structure and Algorithms class',
-    helmet: 'Integrated speech command to a \"smart\" bike helmet built by a team of 9 friends',
+    Nlp: 'Building language models',
+    Ta: 'Held weekly office hours and review sessions for introductory Data Structure and Algorithms class',
+    Helmet: 'Integrated speech command to a \"smart\" bike helmet built by a team of 9 friends',
     unknown: 'Something is wrong, project doesn\'t exist'
   };
   
   const colors = {
```

Comment:
(Optional) I think it might make sense to merge details and colors, so we can access both information for each key.

ex:

const projectKeyToData = {
  NLP: createProjectData('Building ...', '#6ccfe0'),
  ...
};

Reply:
This refactoring makes the function more concise. I created the helper function createProjectData accordingly.